homeassistant/components/smlight/config_flow.py

```diff
@@ -139,10 +139,6 @@ async def async_step_reauth(
             self.context["entry_id"]
         )
         host = entry_data[CONF_HOST]
-        self.context["title_placeholders"] = {
-            "host": host,
```

Comment:
`host` is still used in strings.json for reauth flow. Will need update there also.

Reply:
Thanks, I missed that part, fixed in https://github.com/home-assistant/core/pull/125359/commits/4832e668ab93a2d040e417c4259d43b4c6733100

![reconfig](https://github.com/user-attachments/assets/83bf3039-38b0-47ca-b27c-b8b43322fee0)